e2e/tests/auth.setup.ts

```diff
@@ -8,5 +9,6 @@ setup('authenticate', async ({ page, user }) => {
   await page.getByLabel('Password').fill(user.password)
   await page.getByRole('button', { name: 'Sign in' }).click()
 
+  expect(await page.title()).toBe('Approved Premises')
```

Comment:
Would it be clearer if we wrap this up in an function which is named to express its purpose e.g.

`assertThatLoginSucceeded()`